respa/urls.py

```diff
@@ -50,3 +50,8 @@
     ])
 if settings.DEBUG:
     urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
+
+if settings.DEBUG_TOOLBAR:
```

Comment:
Another pattern is to have

`if "debug_toolbar" in INSTALLED_APPS:`

The reason is that the logic to include or omit debug toolbar is covered in _settings.py_, so we just need to check if it has been included in the installed apps. For example, there's an additional check that `DEBUG` is `True`, which is omitted in _urls.py_.

Reply:
That is a valid point. I made the change.